### PR TITLE
starting to add docs

### LINKED ISSUE #14: Need to have copy of FindNetCDF.cmake and any other necessary cmake files
See https://github.com/NOAA-EMC/NCEPLIBS/issues/128

### PR DESCRIPTION
**Pull request by edwardhartnett**
Part of #15 

Before finishing this we need to resolve #14 and #10.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# This is the main CMake file for NCEPLIBS-wrf_io.
+#
+# Rahul Mahajan
```
Comment (aerorahul) [inline CMakeLists.txt]: Please remove names. 
There is no I. 
git-blame should be enough to identify the contributions.

File: docs/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# This is the CMake file for building the docs directory of NCEPLIBS-wrf_io.
+#
+# Ed Hartnett 10/15/20
+
+IF(ENABLE_DOCS)
+
+  # Create doxyfile.
+  SET(abs_top_srcdir "${CMAKE_SOURCE_DIR}")
+  CONFIGURE_FILE(${CMAKE_CURRENT_SOURCE_DIR}/Doxyfile.in ${CMAKE_CURRENT_BINARY_DIR}/Doxyfile @ONLY)
```
Comment (aerorahul) [inline docs/CMakeLists.txt]: Lower case for cmake function calls

File: docs/CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+  # Build documentation with target all.
+  ADD_CUSTOM_TARGET(doc ALL
```
Comment (aerorahul) [inline docs/CMakeLists.txt]: Lower case

File: docs/CMakeLists.txt [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# This is the CMake file for building the docs directory of NCEPLIBS-wrf_io.
+#
+# Ed Hartnett 10/15/20
+
+IF(ENABLE_DOCS)
+
+  # Create doxyfile.
+  SET(abs_top_srcdir "${CMAKE_SOURCE_DIR}")
```
Comment (aerorahul) [inline docs/CMakeLists.txt]: Lower case

**Review by aerorahul — CHANGES_REQUESTED** [review]
See comments

File: CMakeLists.txt (hunk 1 above)
Comment (edwardhartnett) [reply CMakeLists.txt]: Strongly disagree.

Comment (aerorahul) [reply CMakeLists.txt]: Well, please remove my name.

File: docs/CMakeLists.txt (hunk 2 above)
Comment (edwardhartnett) [reply docs/CMakeLists.txt]: Done.

File: docs/CMakeLists.txt (hunk 3 above)
Comment (edwardhartnett) [reply docs/CMakeLists.txt]: Done.

File: docs/CMakeLists.txt (hunk 4 above)
Comment (edwardhartnett) [reply docs/CMakeLists.txt]: Done.

File: CMakeLists.txt (hunk 1 above)
Comment (edwardhartnett) [reply CMakeLists.txt]: As is customary with science projects we will allow each contributor to add their name to the finished product. 

One purpose of the list of authors in a function is that it is the list of people who take responsibility for that function.

Tracking authorship has benefit besides fairness and historical record. Programmers crave legacy and too often this craving is satisfied by the creation of unneeded code. This adds to project complexity and costs. 

Instead, we will allow everyone to take credit for all the needed code they have contributed, and remove unneeded code. In this way, programmers are encouraged to build legacy as they should - by make valid contributions to necessary code.

Given all this, do you still want me to remove your name?

Comment (aerorahul) [reply CMakeLists.txt]: I take ownership of the code I write. And that is why I am responding to every issue and PR that was my work. 
I do not crave for credit. And I despise legacy where authors like to add their names for every single line of contributions. 
Yes, I wish to remove my name.

Comment (edwardhartnett) [reply CMakeLists.txt]: As you wish.

**Review by aerorahul — APPROVED** [review]
Thanks!

**Pull request merged**